src/hooks/useI18n.js

```diff
@@ -1,4 +1,19 @@
 import { useContext } from 'preact/hooks'
 import { I18nContext } from 'contexts'
+import { getAppLanguage } from 'utils'
 
-export const useI18n = () => useContext(I18nContext)
+export const useI18n = lang => {
+  const banana = useContext(I18nContext)
+  banana.setLocale(getAppLanguage())
```

Comment:
do we foresee the performance issue when reseting the locale everytime it calls the useI18n?

Reply:
Yes, let's not do that.

https://github.com/wikimedia/banana-i18n/blob/7259156f439fefe9e1239888585ecec6a02699ff/src/index.js#L31

Reply:
Update in https://github.com/wikimedia/wikipedia-kaios/pull/196/commits/caa953a7f27a135d9b3cbfa777a8f074a0f5b3ee

It's still needed but only execute when changing the app language